app/src/main/java/com/checkin/app/checkin/home/fragments/UserLocationFragment.kt

```diff
@@ -0,0 +1,114 @@
+package com.checkin.app.checkin.home.fragments
+
+import android.content.Context
+import android.os.Bundle
+import android.text.Editable
+import android.view.View
+import android.widget.EditText
+import android.widget.ImageView
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.Observer
+import butterknife.BindView
+import butterknife.OnClick
+import butterknife.OnTextChanged
+import com.airbnb.epoxy.EpoxyRecyclerView
+import com.checkin.app.checkin.R
+import com.checkin.app.checkin.data.resource.Resource
+import com.checkin.app.checkin.home.epoxy.cityLocationModelHolder
+import com.checkin.app.checkin.home.epoxy.currentLocationModelHolder
+import com.checkin.app.checkin.home.viewmodels.HomeViewModel
+import com.checkin.app.checkin.home.viewmodels.UserLocationViewModel
+import com.checkin.app.checkin.misc.fragments.BaseFragment
+import com.checkin.app.checkin.utility.Constants
+
+class UserLocationFragment : BaseFragment() {
+    override val rootLayout = R.layout.fragment_user_location_switch
+    val viewModel: UserLocationViewModel by viewModels()
+    val homeViewModel: HomeViewModel by activityViewModels()
+    @BindView(R.id.et_user_location)
+    internal lateinit var etUserLocation: EditText
+
```

Comment:
Don't waste lines in between View fields... Fix that in all the files (like holder)